generators/server/templates/src/main/java/package/web/rest/_AccountResource.java

```diff
@@ -200,17 +201,20 @@ public UserDTO getAccount(Principal principal) {
                 return userDTO;
             } else {
                 // Allow Spring Security Test to be used to mock users in the database
-                return Optional.ofNullable(userService.getUserWithAuthorities())
-                    .map(UserDTO::new)
-                    .orElseThrow(RuntimeException::new);
+                Optional<User> user = Optional.ofNullable(userService.getUserWithAuthorities());
```

Comment:
`.orElseThrow(() -> new DefaultException("User could not be found"));` would be better

Reply:
Same as above, I want to remove the `.orElseThrow` method for Sonar

Reply:
No the return param is used here and Sonar doesn't complain

Reply:
oh yes you are correct

Reply:
`.orElseThrow(() -> ProblemBuiler.withTitle("User could not be found").build());` and DefaultException class can be removed (the method then throws DefaultProblem)

Reply:
.orElseThrow(() -> ProblemBuilder.withTitle("User could not be found").build());